src/api/ShlinkApiClient.ts

```diff
@@ -32,6 +32,10 @@ import {
   replaceAuthorityFromUri,
 } from './utils';
 
+export type ShlinkClientOptions = {
```

Comment:
Maybe let's call this `ShlinkApiClientOptions`?

Reply:
Yep, I'm not sure why I didn't do that in the first place.